docs/advanced/storagenetwork.md

```diff
@@ -290,6 +290,101 @@ metadata:
+```
+
+Follow the steps below to test the communication between Longhorn pods.
+
+4.1 Get the pod Storage Network IP from the above step.
+
+```
+Example
+
+instance-manager-r-43f1624d14076e1d95cd72371f0316e2
+Storage Network IP: 10.0.16.8
+
+instance-manager-e-ba38771e483008ce61249acf9948322f
+Storage Network IP: 10.0.16.14
+```
+
+4.2 Login to those two pods.
+
+From the command `ip addr`, you should observe two ip addresses same as the pod annotations, one for pod network and another for Storage Network.
+
+```
+Example:
+
+$ kubectl exec -i -t -n longhorn-system instance-manager-e-ba38771e483008ce61249acf9948322f -- /bin/sh
+
+$ ip addr
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+    inet 127.0.0.1/8 scope host lo
+       valid_lft forever preferred_lft forever
+    inet6 ::1/128 scope host
+       valid_lft forever preferred_lft forever
+3: eth0@if2277: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1450 qdisc noqueue state UP group default
+    link/ether 0e:7c:d6:77:44:72 brd ff:ff:ff:ff:ff:ff link-netnsid 0
+    inet 10.52.6.146/32 scope global eth0
+       valid_lft forever preferred_lft forever
+    inet6 fe80::c7c:d6ff:fe77:4472/64 scope link
+       valid_lft forever preferred_lft forever
+4: lhnet1@if2278: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default
+    link/ether fe:92:4f:fb:dd:20 brd ff:ff:ff:ff:ff:ff link-netnsid 0
+    inet 10.0.16.14/20 brd 10.0.31.255 scope global lhnet1
+       valid_lft forever preferred_lft forever
+    inet6 fe80::fc92:4fff:fefb:dd20/64 scope link
+       valid_lft forever preferred_lft forever
+
+$ ip route
+default via 169.254.1.1 dev eth0
+10.0.16.0/20 dev lhnet1 proto kernel scope link src 10.0.16.14
+169.254.1.1 dev eth0 scope link
+```
+
+4.3 Start a simple http server in one pod.
+
+```
+Example:
+
+$ python3 -m http.server 8000 --bind 10.0.16.14 (replace with your pod Storage Network IP)
+```
+
+:::note
+
+Explicitly bind the simple http server to Storage Network IP.
+
+:::
+
+4.4 Test the http server in another pod:
+
+```
+Example:
+
+From instance-manager-r-43f1624d14076e1d95cd72371f0316e2 (IP 10.0.16.8)
+
+$ curl http://10.0.16.14:8000
+```
+
+4.5 Troubleshooting
+
+When Storage Network works correctly, the `curl` will return a list of files under the http server.
+
+If not, troubleshooting your external network, for example:
+
+- The Storage Network related physical Nics on Harvester nodes are added to the same VLAN in the external switchs.
+
+- The external switchs are correctly connected and configured.
```

Comment:
```suggestion
When the storage network is functioning correctly, the `curl` command returns a list of files on the HTTP server.

4.5 (Optional) Troubleshoot issues.

The storage network may malfunction because of issues with the external network, such as the following:

- Physical NICs (installed on Harvester nodes) that are associated with the storage network were not added to the same VLAN in the external switches.

- The external switches are not correctly connected and configured.
```

Reply:
I moved the info about the curl command to step 4.4 (after the code block). Step 4.5 should focus on troubleshooting. I changed the wording in the list to describe *issues* that users should watch out for.

Reply:
Great, thanks.